images/openshift-ci/Dockerfile

```diff
@@ -1,7 +1,7 @@
 # This Dockerfile is used by openshift CI
 # It builds an image containing crc and nss-wrapper for remote deployments, as well as the google cloud-sdk for nested GCE environments.
 
-FROM registry.svc.ci.openshift.org/openshift/release:golang-1.15 AS builder
+FROM registry.svc.ci.openshift.org/openshift/release:golang-1.16 AS builder
```

Comment:
@cfergeau Since `registry.svc.ci.openshift.org` is now deprecated so better to use https://github.com/openshift/installer/commit/f821cbe00afa0ec4721bfeed7e418d1a59ee842e#diff-d0a313eeccf4bb6517283fba06a12fd97ebc657d01cbee6e2bb834f4af151748R4 something like this.